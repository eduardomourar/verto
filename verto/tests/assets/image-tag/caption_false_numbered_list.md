1.  Lipsum

    *   Ichi
    *   Ni
    *   San

2.  Lorem

3.  Ipsem

    {image file-path="http://placehold.it/350x150" source="https://placehold.it/" hover-text="This is hover text" alignment="left"}
