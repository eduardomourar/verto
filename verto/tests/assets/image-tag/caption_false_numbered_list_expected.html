<ol>
<li>
<p>Lipsum</p>
<ul>
<li>Ichi</li>
<li>Ni</li>
<li>San</li>
</ul>
</li>
<li>
<p>Lorem</p>
</li>
<li>
<p>Ipsem<div>
<img src="http://placehold.it/350x150" title="This is hover text" />
<p><a href="https://placehold.it/">Source</a></p>
</div>
</p>
</li>
</ol>
